pkgs/by-name/vi/visual-hexdiff/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  fetchpatch,
+  ncurses,
+}:
+stdenv.mkDerivation {
+  pname = "visual-hexdiff";
+  version = "0.0.53";
+
+  src = fetchurl {
+    url = "mirror://ubuntu/pool/universe/h/hexdiff/hexdiff_0.0.53.orig.tar.gz";
+    sha256 = "sha256-M1bmkW63pHlfl9zNWEq0EGN1rpVGo+BTUKM9ot4HWqo=";
+  };
+# Fix compiler error that wants a string literal as format string for `wprintw`
+  prePatch = ''
+    substituteInPlace sel_file.c \
+      --replace 'wprintw(win, txt_aide_fs[foo]);' 'wprintw(win, "%s", txt_aide_fs[foo]);'
+  '';
+
+  patches = [
+    # Some changes the debian/ubuntu developers made over the original source code
```

Comment:
That doesn't explain anything about what the changesare

Reply:
added a link to the associated changelog